src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  booksList: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state) => {
+      state.booksList = [];
+    },
+    removeBook: (state) => {
+      state.booksList = [];
+    },
```

Comment:
- [ ] I am not seeing any actions here. Please add the code to add and remove the book.